integration-tests/tests/setup/mod.rs

```diff
@@ -284,4 +288,63 @@ impl VenearTestWorkspace {
             .await?
             .json()?)
     }
+
+    pub async fn get_venear_unlock_timestamp(
+        &self,
+        account_id: &AccountId,
+    ) -> Result<Timestamp, Box<dyn std::error::Error>> {
+        Ok(self
+            .sandbox
+            .view(account_id, "get_venear_unlock_timestamp")
+            .args_json(json!({}))
+            .await?
+            .json()?)
+    }
+
+    pub async fn get_venear_locked(
+        &self,
+        account_id: &AccountId,
+    ) -> Result<WrappedBalance, Box<dyn std::error::Error>> {
+        Ok(self
+            .sandbox
+            .view(account_id, "get_venear_locked_balance")
+            .args_json(json!({}))
+            .await?
+            .json()?)
+    }
+
+    pub async fn get_venear_pending(
+        &self,
+        account_id: &AccountId,
+    ) -> Result<WrappedBalance, Box<dyn std::error::Error>> {
+        Ok(self
+            .sandbox
+            .view(account_id, "get_venear_pending_balance")
+            .args_json(json!({}))
+            .await?
+            .json()?)
+    }
+
+    pub async fn get_lockup_update_nonce(
+        &self,
+        account_id: &AccountId,
+    ) -> Result<u64, Box<dyn std::error::Error>> {
+        Ok(self
+            .sandbox
+            .view(account_id, "get_lockup_update_nonce")
+            .args_json(json!({}))
+            .await?
+            .json()?)
+    }
+}
+
+pub fn into_wrapped(amount: NearToken) -> WrappedBalance {
+    WrappedBalance::from(amount.as_yoctonear())
+}
```

Comment:
They are the same. We should get rid of `WrappedBalance`

Reply:
Resolved in https://github.com/fastnear/house-of-stake-contracts/pull/11/commits/1c4d6a82f78a417d875b282f179b3d2e3db369cf